transform/transformer.go

```diff
@@ -233,12 +235,24 @@ func (t *LogTransformer) transformDiff() {
 	t.LogContent = strings.Join(transformedLines, "\n")
 }
 
-// truncate to avoid Message:Body is too long (maximum is 65536 characters)
+// truncate to avoid Message:Body is too long (maximum is set per VCS)
 func (t *LogTransformer) truncate() {
+	var maxCommentLength int
```

Comment:
well setting a default here, would prevent any nilpointer exception. now if we add a new provider it can happen again. however I will merge now, follow up prs can be done 
```
maxCommentLength := GithubMaxCommentLength
```